content/development/faq.html

```diff
@@ -62,7 +62,9 @@ <h3>Do you have a mailing list for developers?</h3>
 <a name="irc"></a>
 <h3>Do you have an IRC channel?</h3>
 
-<p>There are multiple <a href="/community/irc">IRC Channels</a>. Currently, all of the listed ones are on the irc.freenode.net network. [<a href="#top">top</a>]</p>
```

Comment:
maybe this question should be reworded in a more general way, as some channels (haiku-es and haiku-ru) are in fact on Telegram